samples/msal-node-samples/VanillaJSTestApp/app/routes/authCode.js

```diff
@@ -0,0 +1,35 @@
+/*
+*  Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+*  See LICENSE in the source repository root for complete license information.
+*/
+
+module.exports = function(app, clientApplication, msalTokenCache) {
+    app.get('/', (req, res) => {
+        const authCodeUrlParameters = {
```

Comment:
This will eventually also be factored into an importable file, right?

Reply:
I'd missed these requests objects for authCode. I just went over everything with a fine-toothed comb, hopefully I didn't miss anything else after the refactor. Thanks for catching these!